staging/src/k8s.io/apiserver/pkg/server/config_selfclient_test.go

```diff
@@ -62,8 +79,27 @@ func TestLoopbackHostPort(t *testing.T) {
 	if ip := net.ParseIP(host); ip == nil || !ip.IsLoopback() || ip.To4() != nil {
 		t.Fatalf("expected IPv6 host to be loopback, got %q", host)
 	}
-
 	if port != "443" {
 		t.Fatalf("expected 443 as port, got %q", port)
 	}
 }
+
+func isIPv6LoopbackSupported() (bool, bool, error) {
```

Comment:
If you change it to be like this, it's self-documenting:

`func supportedLoopbacks() (ipv4 bool, ipv6 bool, err error) {`

Reply:
good suggestion, done, @lavalamp

Reply:
Thanks for fixing--as evidence that it was a good change, clearly I had misunderstood what the bools meant, haha!